youtube_dl/extractor/soundcloud.py

```diff
@@ -559,7 +559,7 @@ def _real_extract(self, url):
 class SoundcloudPagedPlaylistBaseIE(SoundcloudIE):
     def _extract_playlist(self, base_url, playlist_id, playlist_title):
         COMMON_QUERY = {
-            'limit': 80000,
+            'limit': 2000,
```

Comment:
should be set to 200 to reflect official SC documentation. It may work but it could also lead to problems again in the future. Maybe there is a side effect using such a huge number when trying to download huge masses (speaking of 1000s and 10000s of tracks) which will trigger error earlier.

Reply:
Works for me. Updated to use 200 instead.